src/index.js

```diff
@@ -1,10 +1,23 @@
 const {
   BaseKonnector,
   requestFactory,
-  saveFiles,
-  addData
+  signin,
+  scrape,
+  saveBills,
+  log
 } = require('cozy-konnector-libs')
-const request = requestFactory({ cheerio: true })
+const request = requestFactory({
+  // the debug mode shows all the details about http request and responses. Very usefull for
+  // debugging but very verbose. That is why it is commented out by default
+  // debug: true,
+  // activates [cheerio](https://cheerio.js.org/) parsing on each page
+  cheerio: true,
+  // If cheerio is activated do not forget to deactivate json parsing (which is activated by
+  // default in cozy-konnector-libs
+  json: false,
+  // this allows request-promise to keep cookies between requests
+  jar: true
+})
 
 const baseUrl = 'http://books.toscrape.com'
```

Comment:
`const absUrl = relpath => `http://books.toscrape.com/${relpath}`?
Or even better, we could ensure `request.defaults({baseUrl: ...})` works.